README.md

```diff
@@ -2,167 +2,151 @@
 ```
 in the root folder of the package.
 
 
 ### C. Path
-Whatever method you choose, if you wish to access the modules located in **PreREISE** from anywhere on your machine, do:
+Whatever method you choose, if you wish to access the modules located in **PreREISE**
+from anywhere on your machine, do:
 ```bash
 pip install .
 ```
-in the root folder of your package or alternatively, setup the `PYTHONPATH` global variable.
+in the root folder of your package or alternatively, setup the `PYTHONPATH` global
+variable.
 
 
 ## 2. Gather Data for Simulation
 This module aims at gathering the required data for building the profiles
 
 ### Profile Naming Convention
-We call the time series profiles directly generated from mathematical/heuristic
-models without any scaling (as we do in different scenarios) as 'raw' profiles.
-The name of a raw profile in the simulation framework complies with following
-rule:
-
-[***Network***]\_[***Profile Type***]\_v[***M***].[***D***].[***C***]
-* ***Network*** specifies the network that a profile applies. It could be an
-interconnection or a combination of two interconnections. Available options
-are: 'texas', 'western', 'eastern', 'usa' and 'texaswestern'.
-* ***Profile Type*** specifies the type of a profile. Available options
-currently are: '*wind*', '*solar*', '*hydro*', '*demand*'.
-* ***M*** is an integer that specifies the methodology used to generate the
-corresponding profile. Details for these methodologies are presented in the
-tables below.
-* ***D*** is an integer that specifies the dimensionality of the corresponding
-profile, i.e., #hours * #generators or #loadzones.
-* ***C*** is an integer that specifies other changes to the corresponding
-profile except for M and D, such as fixing a bug, etc.
-
----
-##### Note:
-1. ***M***, ***D*** and ***C*** are independent of each other and the meaning
-of each value is defined globally for a particular ***Network*** and ***Profile
-Type***. For example, the eastern_wind_v1.2.3 and eastern_wind_v2.2.4 profiles
-would be generated by different methodologies, proceeded with different
-fixes/changes but have exactly same dimensionality.
-2. Profile names for combinations of interconnections are defined according to
-the latest profiles among the interconnections it combines.
----
+We call the time series profiles directly generated from mathematical/heuristic models
+without any scaling (as we do in different scenarios) as 'raw' profiles.
+
+The name of a raw profile in the simulation framework must be of the form:
+
+[**type**]\_v\[**D**],
+
+where **type** specifies the profile type  ('*demand*', '*hydro*', '*solar*', '*wind*')
+and **D** refers to the date the file has been created. Month (3 letters, e.g., Jan) and
+year (4 digits, e.g., 2021) are used for the date. If two profiles of the same **type**
+are created the same month of the same year, the letter **b** will then be appended,
+e.g, ***demand_Jan2021.csv*** and ***demand_Jan2021b.csv***.
```

Comment:
We keep appending `b` when we have more such conflicts, ***demand_Jan2021bb.csv***?

Reply:
Also, should it be ***demand_vJan2021.csv*** and ***demand_vJan2021b.csv***?

Reply:
> We keep appending `b` when we have more such conflicts, ***demand_Jan2021bb.csv***?

I will rephrase it so it is clear that it should be b then c and so forth.

Reply:
Done